static/js/components/forms/RegisterEmailForm.js

```diff
@@ -13,14 +13,22 @@ import { emailValidationShape } from "../../lib/form"
 const emailValidation = yup.object().shape({
   email:     emailValidationShape,
   recaptcha: SETTINGS.recaptchaKey
-    ? yup.string().required("Please verify you're not a robot")
+    ? yup
+      .string()
+      .nullable()
+      .required("Please verify you're not a robot")
```

Comment:
Why add `nullable()`, I think that would be overriden by `required()`? 
>mixed.required(message?: string | function): Schema
>Mark the schema as required. All field values apart from undefined and null meet this requirement.

Reply:
It was a half-hearted attempt to fix #345 but seeing as it's probably not a correct fix and I didn't write a test for it, I'm just going to remove it for now.